src/main/webapp/dev/js/main/adminSearch.es6

```diff
@@ -144,4 +145,56 @@ $(document).ready(() => {
     $('#btn-collapse-all-students').on('click', () => {
         adminSearchCollapseAllStudents();
     });
+
+    $('.openEmailApplicationDefaultValues').submit((e) => {
+        e.preventDefault();
+        const $entry = $(e.target);
+        const studentEmail = $entry.parentsUntil('.fslink_student').find('input[name=studentEmail]').val();
+        const subjectType = $entry.find('input[name=subjectType]').val();
+        const courseName = $entry.find('input[name=courseName]').val();
+        const courseId = $entry.find('input[name=courseId]').val();
+        const studentName = $entry.find('input[name=studentName]').val();
+        const sessionStatus = $entry.find('input[name=sessionStatus]').val();
+        const sessionName = $entry.find('#feedback-session-name').text();
+        const sessionNameWithoutDate = sessionName.substring(0, sessionName.lastIndexOf(' ['));
+
+        const relatedLink = encodeURIComponent($entry.find('input[name=relatedLink]').val());
+        const googleSignup = encodeURIComponent('https://accounts.google.com/NewAccount');
+        const supportEmail = `${Const.TEAMMATES_EMAIL_ADDRESS}`;
+
+        const bodyHeader = `Hello ${studentName},`;
+        const bodyFooter =
+                '%0D%0A%0D%0AIf you encounter any problems when using the system,'
```

Comment:
@wkurniawan07 any suggestions on the variable name? :sweat_smile:

Reply:
I got a question on why u chose `%0D%0A` though? Is it because browsers always encode newlines as `%0D%0A` regardless of platform? That seems to be the case but I think it will be nice to include an official reference and I do think there should be some comments in the code indicating why because it is an implementation detail.

Since that is a CLRF, I'm also wondering if there is a better approach to encode urls like browsers so we can avoid re-implementing the implementation detail and having to write comments.

Can you check if that is possible since encodeURIComponent does not translate `\n` to `%0D0A`. Either way, it will be more readable if u choose something like `const clrf = encodeURIComponent('\r\n')`.

Reply:
Done